src/components/Borrow/BorrowInfo/BorrowInfo.tsx

```diff
@@ -0,0 +1,186 @@
+          <Grid
+            container
+            item
+            className={classes.infoPosition}
+            justifyContent='space-between'
+            alignItems='center'>
+            <Typography className={classes.positionTitle}>Collateral amount (xUSD):</Typography>
+            <Typography className={classes.positionValue}>
+              {'$'}
+              {collateralAmount}
+            </Typography>
+          </Grid>
+          <Grid
+            container
+            item
+            className={classes.infoPosition}
+            justifyContent='space-between'
+            alignItems='center'>
+            <Typography className={classes.positionTitle}>Debt amount:</Typography>
+            <Typography className={classes.positionValue}>
+              {'$'}
+              {debtAmount}
+            </Typography>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid className={classes.wrapper}>
+        <Typography className={classes.infoTitle}> General info</Typography>
+        <Grid className={classes.infoWrapper}>
+          <Grid
+            container
+            item
+            className={classes.infoPosition}
+            justifyContent='space-between'
+            alignItems='center'>
+            <Typography className={classes.positionTitle}>Collateral:</Typography>
+            <Typography className={classes.positionValue}>{collateral}</Typography>
+          </Grid>
+          <Grid
+            container
+            item
+            className={classes.infoPosition}
+            justifyContent='space-between'
+            alignItems='center'>
+            <Typography className={classes.positionTitle}>Borrowed asset:</Typography>
+            <Typography className={classes.positionValue}>{borrowed}</Typography>
+          </Grid>
+          <Grid
+            container
+            item
+            className={classes.infoPosition}
+            justifyContent='space-between'
+            alignItems='center'>
+            <Typography className={classes.positionTitle}>Limit:</Typography>
+            <Typography className={classes.positionValue}>
+              {'$'}
+              {limit}
+            </Typography>
+          </Grid>
+          <Grid
+            container
+            item
+            className={classes.infoPosition}
+            justifyContent='space-between'
+            alignItems='center'>
+            <Typography className={classes.positionTitle}>Reserve:</Typography>
+            <Typography className={classes.positionValue}>
+              {'$'}
+              {reserve}
+            </Typography>
+          </Grid>
+          <Grid
+            container
+            item
+            className={classes.infoPosition}
+            justifyContent='space-between'
+            alignItems='center'>
+            <Typography className={classes.positionTitle}>Collateral address:</Typography>
+            <Grid container item direction='row' alignItems='center' className={classes.copy}>
```

Comment:
I think it mighty be a good idea to create a new component and use it also in swap info